content/en/docs/tasks/tools/_index.md

```diff
@@ -14,6 +14,11 @@ cluster resources, and view logs.
 See [Install and Set Up kubectl](/docs/tasks/tools/install-kubectl/) for information about how to
 download and install `kubectl` and set it up for accessing your cluster.
 
+<form action="/docs/tasks/tools/install-kubectl/">
+  <input type="submit" class="btn btn-primary" value="Install and Set Up kubectl" />
```

Comment:
Hi @nate-double-u . Some feedback (I may have misunderstood your intention):
What is the advantage of adding the "form" and "button"? The same link is right above?
Should the button provide a hint as to what is happening when you hit the button?
I'd rather not add HTML to the page.

Reply:
If we did use a button, I'd prefer `<button>` to `<input>`.

Reply:
I added the button as a way to help folks get to the info they're looking for a little quicker (based on a [suggestion in the issue](https://github.com/kubernetes/website/issues/23354#issuecomment-700295045)), and tried to keep it in line with other button-links on the site (https://kubernetes.io/docs/home/#understand-the-basics)

I agree that the text could be clearer about what's going to happen next.

I also agree about not wanting to add HTML to the page, but I couldn't find a button shortcode.
Should I remove the buttons altogether?

Reply:
The buttons are a bold UI statement. I would follow
the pattern of the other site buttons by permitting the text to be localized (add an
Aria label or not?). I'd like to see a tooltip or some hint that this button is forwarding
me to another page (not installing/starting up the tool).

Reply:
I've added the Aria labels, and have changed the wording of each of the buttons so that it is more obvious what they will do.